examples/gatsby-builder-example-no-plugin/README.md

```diff
@@ -0,0 +1,94 @@
+<p align="center">
```

Comment:
needs a new readme (I know you have it planned, just adding comment here to track it)